src/test_workflow/bwc_test/bwc_test_runner.py

```diff
@@ -46,9 +46,9 @@ def run(self) -> TestSuiteResults:
                         test_results = test_suite.execute_tests()
                         all_results.append(component.name, test_results)
                     else:
-                        logging.info(f"Skipping bwc-tests for {component.name}, as it is currently not supported")
+                        logging.info(f"Skipping bwc-tests for {component.name}, as it is currently not supported according to the test manifest.")
```

Comment:
What does `according to the test manifest` mean here.

Reply:
```
logging.info(f"Skipping bwc-tests for {component.name}, as the component in the test manifest has no bwc-tests section")
```

Reply:
This log info means the component is declared in the manifest but it's not supported with bwc test. (e.g. ml-commons)
The one below means the component in our distribution manifest is not declared in the test manifest. e.g. (common-utils)